pkg/autoscaler/autoscaler_test.go

```diff
@@ -495,7 +495,35 @@ type linearSeries struct {
 
 type mockReporter struct{}
 
-func (r *mockReporter) Report(m Measurement, v float64) error {
+func (r *mockReporter) ReportDesiredPodCount(v int64) error {
```

Comment:
These could use comments

Reply:
Much like the comments on the non-mock Reporter these functions are exported and could use comments on them.

i.e
// ReportDesiredPodCount .....

Reply:
Done.